### PR TITLE
Finished front end work for the abstract deck class.

### LINKED ISSUE #17: implement abstract class for second deck

### PR DESCRIPTION
**Pull request by barrieca**
This closes #17

### CODE REVIEW

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -60,6 +89,25 @@
     game = data;
 });
 
+$("#country").change(function(){
+    country = $(this).val();
+    chooseDeck($(this).val());
+});
+
+function chooseDeck(d){
```
Comment (parkj6) [inline src/main/java/views/AcesUp/AcesUp.flt.html:97]: is this the button for the "choose country?" awesome. good job dude.

**Comment by parkj6** [comment]
Everything looks good from this end. let me run the ninja script and give you more feedback.

**Comment by barrieca** [comment]
Yeah, that is a html select tag that chooses the deck.

**Comment by barrieca** [comment]
Oops. Accidentally closed.

**Pull request merged**